e2e/tests/functional/search.e2e.spec.js

```diff
@@ -139,6 +139,45 @@ test.describe("Search Tests @unstable", () => {
         await expect(searchResults).toHaveText(folderName);
     });
 
+    test('Search results are debounced @couchdb', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/6179'
+        });
+        //Go to baseURL
+        await page.goto("./", { waitUntil: "networkidle" });
```

Comment:
Looks like `createObjectsForSearch()` is doing this also. Let's move this line to a `beforeEach()`, remove it from other tests in this suite, and remove the `page.goto()` from `createObjectsForSearch()`.